tests/e2e/sample-applications/virtual-machines/cirros-test/cirros-ds.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: cdi.kubevirt.io/v1beta1
+    kind: DataVolume
+    metadata:
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: ""
+        cdi.kubevirt.io/storage.deleteAfterCompletion: "false"
+      name: cirros
+      namespace: cirros-test
```

Comment:
would make as equal to tests as possible. openshift-virtualization-os-images namespace is used there https://github.com/openshift/oadp-operator/blob/master/tests/e2e/virt_backup_restore_suite_test.go#L126